client/src/includes/breadcrumbs.test.js

```diff
@@ -0,0 +1,70 @@
+import initCollapsibleBreadcrumbs from './breadcrumbs';
+
+describe('initCollapsibleBreadcrumbs', () => {
+  it('should do nothing if there is no breadcrumbs container', () => {
+    jest.spyOn(document, 'addEventListener').mockImplementation(() => {});
+    // Set up our document body
+    document.body.innerHTML =
+      '<div>' +
+      '  <span id="username" />' +
+      '  <button id="button" />' +
+      '</div>';
```

Comment:
quick tip, you can use template literals (backticks) to make this easier to write.

```
    document.body.innerHTML = `
      <div>
        <span id="username" />
        <button id="button" />
      </div>`;
```

Reply:
This indeed made it much easier